ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -123,3 +123,7 @@ def batch_normalization(x, mean, variance, offset, scale, variance_epsilon, name
     return x * ivy.astype(inv, x.dtype, copy=False) + ivy.astype(
         offset - mean * inv if offset is not None else -mean * inv, x.dtype
     )
+
+
+def dropout(x, rate, noise_shape=None, seed=None, name=None):
+    return ivy.dropout(x, rate, noise_shape, seed)
```

Comment:
The parameters of the `ivy.dropout` function don't match with the actual parameters accepted by the function, please check the `dropout` function inside `functional/ivy/layers.py` and pass the parameters accordingly. Firstly, the parameter names of the `ivy.dropout` function may not always match with the parameters of the frontend function, but in such cases you can just refer the docs of the tensorflow dropout function and the docstring of the `ivy.dropout` function and understand which parameters have the same meaning. Finally, when you pass the parameters, there are some that are positional-only (the ones before `/`) and some that are keyword-only (the ones after `*`) so please pass them accordingly.